src/commands/hutte/org/list.ts

```diff
@@ -0,0 +1,44 @@
+import { flags, SfdxCommand } from '@salesforce/command';
+
+import { getScratchOrgs, IScratchOrg } from '../../../api';
+import { projectRepoFromOrigin } from '../../../common';
+
+export default class List extends SfdxCommand {
+  public static description = 'list hutte scratch orgs from current repository';
+
+  static requiresProject = true;
+
+  protected static flagsConfig = {
+    includeauth: flags.boolean({
```

Comment:
This could also be named `verbose`.
Salesforce has a similar flag in `sfdx org display`:

```
$ sfdx org display --help
...
--verbose                 Display the sfdxAuthUrl property.
...
```

Reply:
I agree, however `verbose` is a reserved word in SFDX commands which I was not able to set :(

Reply:
Oh, I didn't know that. I'll check if there is another way to use `--verbose`.

Reply:
Maybe using `flags.builtin`?

```javascript
    verbose: flags.builtin({
      description: messages.getMessage('flags.verbose'),
      longDescription: messages.getMessage('flagsLong.verbose'),
    }),

```

Reply:
Awesome! Thank you! It worked, just committed